docs/backends/generic-mysql.rst

```diff
@@ -124,6 +124,13 @@ Enable DNSSEC processing for this backend. Default: no.
 
 Use the InnoDB READ-COMMITTED transaction isolation level. Default: yes.
 
+.. _setting-gmysql-ssl:
+
+``gmysql-ssl``
+^^^^^^^^^^^^^^^^^^
+
```

Comment:
```suggestion
.. versionadded:: 4.2.1
```
Or 4.3.0 if we don't intend on backporting this?

Reply:
I think a backport makes sense, and it saves Kees building custom RPMs, I understand.